### DISCUSSION TITLE
Cloud Function

### DISCUSSION THREAD

**Discussion by kschmott** [mifi/editly#263] [Q&A]
Hello,

Has anyone been able to get this working with a cloud function?

**Comment by bkeepers** [comment]
For now you need gl, which requires xfvb (see https://github.com/stackgl/headless-gl?tab=readme-ov-file#how-can-headless-gl-be-used-on-a-headless-linux-machine).

There are some issues and pull requests to make gl an optional dependency, which as long as you don't need transitions, it might work on a cloud function.
